docs/omero-web-release-process.rst

```diff
@@ -9,6 +9,13 @@ This document describes the release process for `OMERO Web`_.
 The release process uses GitHub actions, make sure that the actions are **active** before pushing any tag.
 
 
+Register CVE
+^^^^^^^^^^^^
+
+As soon as a vulnerability is identified, create a security advisory on `GitHub <https://github.com/ome/omero-web/security/advisories>`_.
+The work to fix the vulnerability will be done using the private copy of `ome/omero-web <https://github.com/ome/omero-web/>`_.
+The release process needs to be adjusted in that case.
```

Comment:
How does the release process need to be adjusted for web?

Reply:
I notice we have a different (out of date) release process for web at https://ome-contributing.readthedocs.io/en/latest/omero-web-release-process.html#omero-web-release (doesn't use bumpversion)

Reply:
The release process is similar, private branch, squash commits, tag and push when ready to public repo

Reply:
> I notice we have a different (out of date) release process for web at https://ome-contributing.readthedocs.io/en/latest/omero-web-release-process.html#omero-web-release (doesn't use bumpversion)

It points to the release process from the README, it mentions ``bump2version``